runtime/common/src/lib.rs

```diff
@@ -71,10 +68,7 @@ macro_rules! decl_common_types {
             frame_system::ChainContext<Runtime>,
             Runtime,
             AllPalletsWithSystem,
-            (
-                zrml_market_commons::migrations::UpdateMarketsForAuthorizedMDM<Runtime>,
-                zrml_prediction_markets::migrations::TransformScalarMarketsToFixedPoint<Runtime>,
-            )
+            ()
```

Comment:
You should either remove this...

```suggestion
```

...or add a trailing comma:

```suggestion
            (),
```